distribution/docs/src/main/resources/_contents/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,480 @@
+====
+
+[WARNING]
+====
+Only one configuration file should exist for any given PID.
+The result of having both a `.cfg` and a `.config` file for the same PID is undefined and could cause the application to fail.
+====
+
+The main purpose of the configuration files is to allow administrators to pre-configure ${branding} without having to use the {admin-console}.
+In order to do so, the configuration files need to be copied to the `<${branding}_HOME>/etc` directory after ${branding} zip has been extracted.
+
+Upon start up, all the `.config` files located in `<${branding}_HOME>/etc` are automatically read and processed.
+Files that have been processed successfully are moved to `<${branding}_HOME>/etc/processed` so they will not be processed again when the system is restarted.
+Files that could not be processed are moved to the `<${branding}_HOME>/etc/failed` directory.
+
+${branding} also monitors the `<${branding}_HOME>/etc` directory for any new `.config` file that gets added.
+As soon as a new file is detected, it is read, processed and moved to the appropriate directory based on whether it was successfully processed or not.
+
+===== Configuring Managed Service Factory Bundles
+
+Services that are created using a Managed Service Factory can be configured using `.config` files as well.
+The configuration files follow a different naming convention however.
+The files must start with the Managed Service Factory PID, be followed by a unique identifier and have a `.config` extension.
+For instance, assuming that the Managed Service Factory PID is `org.codice.${ddf-branding-lowercase}.factory.pid` and two instances of the service need to be configured, files `org.codice.${ddf-branding-lowercase}.factory.pid.uniqueID1.config` and `org.codice.${ddf-branding-lowercase}.factory.pid.uniqueID2.config` should be created and added to `<${branding}_HOME>/etc`.
+
+The unique identifiers used in the file names have no impact on the order in which the configuration files are processed.
+No specific processing order should be assumed.
+Also, a new service will be created and configured every time a configuration file matching the Managed Service Factory PID is added to the directory, regardless of the number used.
+
+These configuration files must also contain a `service.factoryPid` property set to the factory PID (without the sequential number).
+They should not however contain the `service.pid` property.
+
+====== File Format
+
+The basic syntax of the `.config` configuration files is similar to the older `.cfg` files but introduces support for lists and types other than simple strings.
+The type associated with a property must match the `type` attribute used in the corresponding `metatype.xml` file when applicable.
+
+The following table shows the format to use for each property type supported.
+
+[cols="1,2,4" options="header"]
+|===
+
+|Type
+|Format
+|Example
+
+|Service PID
+|`service.pid` = "servicePid"
```

Comment:
This should probably be `service.pid = "servicePid"` (move middle single quote to end)